libs/components/theme/src/lib/styles/themes/modern/_compat/_mixins.scss

```diff
@@ -95,8 +99,14 @@
 
   &:active {
     border-bottom: solid 3px $sky-highlight-color-info;
+    color: var(--sky-text-color-default);
     padding-bottom: $sky-theme-modern-tab-btn-padding-top-bottom - 3px;
   }
+
+  &.sky-btn-tab-disabled {
+    background-color: inherit;
```

Comment:
Should this truly inherit the background color of the parent element, or should it be transparent?

Reply:
Good call. Fixed